domain/domain.py

```diff
@@ -138,7 +136,8 @@ def get_active_attributes(self):
         result = self.ds.engine.execute_query(query)
         if not result:
             raise Exception("No attribute contains erroneous cells.")
-        return set(itertools.chain(*result))
+        # Sort the active attributes to maintain the order of the ids of random variable.
+        return sorted(list(itertools.chain(*result)))
```

Comment:
You can omit the `list` and just have `sorted`.

Reply:
ok